### ISSUE TITLE
v0.9.0 broke the fastlane plugin

### ISSUE THREAD

**Issue by Dids** [voydz/souyuz#32]
v0.9.0 seems to have change how the fastlane plugin spec file sets the version, which breaks the plugin entirely:
```
bundler: failed to load command: fastlane (/Users/vagrant/.rbenv/versions/2.6.3/bin/fastlane)
ArgumentError: Malformed version number string undefined
  /Users/vagrant/.rbenv/versions/2.6.3/lib/ruby/site_ruby/2.6.0/rubygems/version.rb:212:in `initialize'
  /Users/vagrant/.rbenv/versions/2.6.3/lib/ruby/site_ruby/2.6.0/rubygems/version.rb:203:in `new'
  /Users/vagrant/.rbenv/versions/2.6.3/lib/ruby/site_ruby/2.6.0/rubygems/version.rb:203:in `new'
  /Users/vagrant/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/fastlane-2.137.0/fastlane/lib/fastlane/plugins/plugin_update_manager.rb:29:in `block in show_update_status'
  /Users/vagrant/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/fastlane-2.137.0/fastlane/lib/fastlane/plugins/plugin_update_manager.rb:26:in `each'
  /Users/vagrant/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/fastlane-2.137.0/fastlane/lib/fastlane/plugins/plugin_update_manager.rb:26:in `show_update_status'
  /Users/vagrant/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/fastlane-2.137.0/fastlane/lib/fastlane/commands_generator.rb:43:in `ensure in start'
  /Users/vagrant/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/fastlane-2.137.0/fastlane/lib/fastlane/commands_generator.rb:53:in `start'
  /Users/vagrant/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/fastlane-2.137.0/fastlane/lib/fastlane/cli_tools_distributor.rb:119:in `take_off'
  /Users/vagrant/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/fastlane-2.137.0/bin/fastlane:23:in `<top (required)>'
  /Users/vagrant/.rbenv/versions/2.6.3/bin/fastlane:23:in `load'
  /Users/vagrant/.rbenv/versions/2.6.3/bin/fastlane:23:in `<top (required)>'
```

Downgrading back to v0.8.1 fixes it for now.

**Comment by voydz** [comment]
Thanks, I will have a look.

**Comment by voydz** [comment]
this should be fixed with 057ddda2dd49128ae14a88679e92906ffa900477
